roman.py

```diff
@@ -1,13 +1,22 @@
+import re
+
+ROMAN_NUMBERS = (('I',1),('V',5),('X',10),('L',50),('C',100),('D',500),('M',1000))
+
 def roman_to_integer(value):
-    number_dict = {'I':1, "V":5, "X":10, "L":50}
+    if isinstance(value,int):
+        return -1
     total = 0
-    i_sum = value.count("I")
-    v_sum = value.count("V")
-
-    if value == "IV":
-        return 4
-    if value == "IX":
-        return 9
+    pattern = '.*VX'
+    if re.match(pattern,value):
```

Comment:
could put the both conditions together like: `if re.match(pattern,value) or isinstance(value,int):`

Reply:
Yes I can do that for sure.